CONTRIBUTING.md

```diff
@@ -1,24 +1,38 @@
 # How to Contribute
 
-Contributions to libCEED are welcome.
+Contributions to libCEED are encouraged.
 
 Please use a pull request to the appropriate branch ('maint' for
```

Comment:
This references the 'maint' branch which doesn't exist yet. Do we know what version we will start having a 'maint' branch? 1.0?

Reply:
When we have external users. ;-)
I'd like to start with whatever libCEED release goes into the CEED distribution released early next year. Should we delete that bit for now (and remember to add it back when appropriate)?

Reply:
Yes, I think we should delete or comment out that bit for now if its going to be several months before we start having a maint branch.